src/porepy/models/momentum_balance.py

```diff
@@ -41,45 +42,13 @@ class MomentumBalanceEquations(pp.BalanceEquation):
     """Stress on the grid faces. Provided by a suitable mixin class that specifies the
     physical laws governing the stress.
 
-    """
-    mdg: pp.MixedDimensionalGrid
-    """Mixed dimensional grid for the current model. Normally defined in a mixin
-    instance of :class:`~porepy.models.geometry.ModelGeometry`.
-
-    """
-    interfaces_to_subdomains: Callable[[list[pp.MortarGrid]], list[pp.Grid]]
-    """Map from interfaces to the adjacent subdomains. Normally defined in a mixin
-    instance of :class:`~porepy.models.geometry.ModelGeometry`.
-
-    """
-    internal_boundary_normal_to_outwards: Callable[
-        [list[pp.Grid], int], pp.ad.SparseArray
-    ]
-    """Switch interface normal vectors to point outwards from the subdomain. Normally
-    set by a mixin instance of :class:`porepy.models.geometry.ModelGeometry`.
-
     """
     fracture_stress: Callable[[list[pp.MortarGrid]], pp.ad.Operator]
     """Stress on the fracture faces. Provided by a suitable mixin class that specifies
     the physical laws governing the stress, see for instance
     :class:`~porepy.models.constitutive_laws.LinearElasticMechanicalStress` or
     :class:`~porepy.models.constitutive_laws.PressureStress`.
 
-    """
-    basis: Callable[[Sequence[pp.GridLike], int], list[pp.ad.SparseArray]]
-    """Basis for the local coordinate system. Normally set by a mixin instance of
-    :class:`porepy.models.geometry.ModelGeometry`.
-
-    """
-    normal_component: Callable[[list[pp.Grid]], pp.ad.SparseArray]
-    """Operator giving the normal component of vectors. Normally defined in a mixin
-    instance of :class:`~porepy.models.models.ModelGeometry`.
-
-    """
-    tangential_component: Callable[[list[pp.Grid]], pp.ad.SparseArray]
-    """Operator giving the tangential component of vectors. Normally defined in a mixin
```

Comment:
I cannot comment on the relevant line but EquationSystem (below) is defined in the protocol